projects/components/src/stepper/stepper.component.ts

```diff
@@ -96,6 +98,9 @@ export class StepperComponent implements AfterContentInit {
   @Input()
   public orientation: StepperOrientation = StepperOrientation.Horizontal;
 
+  @Input()
+  public buttonDisabled: StepperButtonDisabledExplicitStatus = {};
```

Comment:
because this is not a boolean flag, it is like a map of boolean flags. I think it should be renamed, this name is misleading. Maybe `controlButtonsStatus`  ?

Reply:
Imo, we should flatten it. That way the api would be simpler and we don't need to always pass a new object if only the state for one button changes.